src/containers/RequestBuilder.js

```diff
@@ -315,7 +331,7 @@ export default class RequestBuilder extends Component {
                         callbackMap={this.updateStateMap}
                         // updatePrefetchCallback={PrefetchTemplate.generateQueries}
                         clearCallback={this.clearState}
-                        options={this.state.codeValues}
+                        options={codeValues}
```

Comment:
the options prop doesn't appear to do anything anymore.  It's not really relevant to this PR but we could toss it and any reference to it, since if you follow it through the components, it never gets used.

Reply:
updated